lnrpc/routerrpc/router.proto

```diff
@@ -105,6 +105,13 @@ message SendPaymentRequest {
 
     /// If set, circular payments to self are permitted.
     bool allow_self_payment = 15;
+
+    /**
+    Features assumed to be supported by the final node. All transitive feature
+    depdencies must also be set properly. For a given feature bit pair, either
+    optional or remote may be set, but not both.
+    */
+    repeated uint32 dest_features = 16;
```

Comment:
I am thinking, maybe we also need to introduce these fields on the main payment rpc. Otherwise `lncli` in the keysend pr will need to switch to `routerrpc` already, to be able to pass in the payment addr.

Reply:
i was thinking i made sense to add an experimental feature to routerrpc, but either way is okay with me

Reply:
I am ok with that too, but how is that going to work in `lncli`? Call the same sendpayment logic from a different `lncli` command that is behind a build flag?